draft-ietf-quic-version-negotiation.md

```diff
@@ -387,6 +387,40 @@ lifetime of the connection; endpoints MUST NOT change the version that they
 consider to be in use based on the Version field of long headers as that field
 could be forged by attackers.
 
+# Minimum support
+
+Hosts can support this standard incrementally.
```

Comment:
I think we should expand on this to explain why it might be desirable to only partially support this spec